aviary/utils/fortran_to_aviary.py

```diff
@@ -303,6 +307,14 @@ def set_value(var_name, var_value, value_dict: NamedValues, var_ind=None, units=
         current_value[var_ind] = var_value[0]
         value_dict.set_val(var_name, current_value, units)
     else:
+        if current_value != None and type(current_value[0]) == type(True):
+            # if a variable is defined as boolean but is read in as number, set as boolean
+            if var_value[0] == 1:
+                var_value = ['True']
+            elif var_value[0] == 0:
+                var_value = ['False']
```

Comment:
Should these be `True` and `False` instead of `'True'` and `'False'`?

Reply:
No. All the updates are done as list.

Reply:
Sorry, I didn't mean taking them out of the list, I meant storing them as the boolean values of True/False, instead of the strings 'True'/'False'